benchmarks/distributed/rpc/rl/README.md

```diff
@@ -0,0 +1,64 @@
+# Distributed RPC Benchmark
+
+This tool is used to measure pytorch distributed rpc throughput and latency. This
+is helpful for evaluating the suitability of rpc to applications using pytorch distributed.
+
+In addition to printing measurements, it produces a JSON file.  Users may choose a single argument to provide multiple comma separated entries for (ie: `world_size="10,50,100"`) in which case the JSON file produced can be passed to the plotting repo to visually see how results differ.  In this case, each entry for the variable argument will be placed on the x axis.
+
+The benchmark results comprise of 4 key metrics:
+1. _Agent Latency_ - How long does it take from the time the first `select_action` request in a batch is received from an observer to the time an action is selected by the agent for each request in that batch.  If not using batch, you can think of it as `batch_size=1`.
+2. _Agent Throughput_ - The number of request processed per second for a given batch.  Is literally computed as `(batch_size / agent_latency)`.  If not using batch, you can think of it as `batch_size=1`.
+3. _Observer Latency_ - Time it takes from the moment a `select_action` request is sent out from a single observer to the time the response is received from the agent.  Therefore if not using batch, observer latency is `(agent_latency + communication_between_observer_and_agent_latency)`.  When using batch there will be more variation due to some observer requests being queued in a batch for longer than others depending on what order those requests came into the batch in.
+4. _Observer Throughput_ - Number of requests processed per second for a single observer.  Is literally computed as `(1 / observer_latency)`.  
```

Comment:
I see, this is not a typo. I am not sure if this is correct gramma. Please ignore this comment if the original version is correct. 

observer. Is literally computed as --> batch, which is computed as ??